src/app/views/user/user-settings/user-settings.component.scss

```diff
@@ -39,68 +38,69 @@
         }
 
         .file {
-          width: 85px;
+          cursor: pointer;
           height: 85px;
           opacity: 0;
-          cursor: pointer;
+          width: 85px;
 
           &[disabled] {
             pointer-events: none;
           }
         }
 
         .icon-camera {
-          width: 85px;
+          bottom: 0;
+          display: table;
           height: 85px;
+          left: 0;
           pointer-events: none;
-          display: table;
           position: absolute;
-          top: 0;
-          bottom: 0;
-          left: 0;
           right: 0;
+          top: 0;
+          width: 85px;
 
           .fa-camera {
+            color: rgba(255, 255, 255, 0.7);
             display: table-cell;
-            vertical-align: middle;
             font-size: 35px;
-            color: rgba(255, 255, 255, 0.7);
+            vertical-align: middle;
           }
         }
 
         .user-info-image {
-          pointer-events: none;
-          object-fit: cover;
-          width: 85px;
-          height: 85px;
-          position: absolute;
-          top: 0;
           bottom: 0;
+          height: 85px;
           left: 0;
+          object-fit: cover;
+          pointer-events: none;
+          position: absolute;
           right: 0;
+          top: 0;
+          width: 85px;
         }
       }
 
       .user-profile-subheader-info-name-container {
-        display: flex;
-        flex-wrap: wrap;
+        @include wrap-long-words;
         align-items: center;
+        display: inline-block;
+        flex-wrap: wrap;
         height: 85px;
         margin: 0 auto;
         max-width: 200px;
-        @include wrap-long-words;
+        vertical-align: middle;
       }
 
       .user-profile-subheader-info-success-msg {
+        color: green;
```

Comment:
Properties should be ordered width, margin-top, color

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)